### PR TITLE
Renesas: Smartbond: Update LVGL Coordinates

### PR DESCRIPTION
**Pull request by ioannis-karachalios**
This commit should take care of the changes introduced in #70541.

### CODE REVIEW

**Review by blauret — APPROVED** [review]

**Comment by blauret** [comment]
@andrzej-kaczmarek ping

**Review by andrzej-kaczmarek — APPROVED** [review]

File: drivers/display/display_renesas_lcdc.c [hunk 1]
```diff
@@ -440,7 +440,7 @@ static void display_smartbond_get_capabilities(const struct device *dev,
 	 * bindings.
 	 */
 	capabilities->supported_pixel_formats = DT_INST_PROP(0, pixel_format);
-	capabilities->current_orientation = DISPLAY_ORIENTATION_NORMAL;
+	capabilities->current_orientation = DISPLAY_ORIENTATION_ROTATED_180;
```
Comment (Snevzor) [inline drivers/display/display_renesas_lcdc.c:443]: Is the display rotated 180 degrees or is this a work-around to rotate the input device coordinates?

Fyi there will be some changes regarding rotation and input device: https://github.com/zephyrproject-rtos/zephyr/pull/73474

Comment (ioannis-karachalios) [reply drivers/display/display_renesas_lcdc.c:443]: Thanks for the input. The display panel I am testing with should not be rotated. However, after the changes introduced in #70541 I was no longer able to make it work without playing with the `orientation` property.

Comment (ioannis-karachalios) [reply drivers/display/display_renesas_lcdc.c:443]: I tested with the changes proposed in #73474 and this solves the issues I was observing. As such, this PR should no longer be considered valid. @Snevzor can you confirm that this fix will be part of the upcoming LTS 3.7 release?

Comment (Snevzor) [reply drivers/display/display_renesas_lcdc.c:443]: Hi @ioannis-karachalios , that is up to @faxe1008 I presume. He is writing a unit test for this as well.

Comment (ioannis-karachalios) [reply drivers/display/display_renesas_lcdc.c:443]: OK. This should be considered a bug so I guess it should be included. I converted this PR to `Draft` until more info is available. Once again, thanks for notifying me.

Comment (Snevzor) [reply drivers/display/display_renesas_lcdc.c:443]: Look at this beauty: https://github.com/zephyrproject-rtos/zephyr/pull/73556

**Comment by ioannis-karachalios** [comment]
No need to provide any explicit workaround as there is a bug in the `LVGL` input pointer implementation and that should be fixed in the corresponding PR.

**Pull request closed without merging**